src/checks/ball.js

```diff
@@ -0,0 +1,34 @@
+const _ = require('lodash');
+const ball_data = require('../helpers/ball-data.js');
+const pokemon_list = _.keys(ball_data.legal);
+
+for (let i = 0; i < pokemon_list.length; i++) {
+  const parsed = [];
+  for (let j = 0; j < ball_data.ball_types.length; j++) {
+    const data = parseInt(ball_data.legal[pokemon_list[i]].charAt(j), 8);
+    parsed.push({ball_name: ball_data.ball_types[j], legal: !!(data & 4), ha: !!(data & 2), breedable: !!(data & 1)});
+  }
+  ball_data.legal[pokemon_list[i]] = parsed;
+}
+const pokemon_corrections = {'nidoran-m': 'nidoran♂', 'nidoran-f': 'nidoran♀', nidoran: 'nidoran♀'};
+const ball_corrections = {poké: 'poke'};
```

Comment:
These corrections are only needed for the IRC bot; I don't think we need them if we're operating on parsed pk6 data, since people won't do things like mistyping "Poké" as "Poke".